controller/test-files/codebase/show/ok_without_auth.golden

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Sure, why not. 

> My concern is, if this method is altered, we may have unexpected test failures or even worse, test successes. WDYT ?

Well, the `compareWithGolden` function is part of the `controller_test` package. Do we really want
 to write tests for our test helpers? But sure, I can add such a test.